bin/main.rb

```diff
@@ -1,3 +1,32 @@
 #!/usr/bin/env ruby
 
-puts 'Hello world!'
+puts 'Wellcome to Roby Tic Tac Toe!'
+
+puts 'Enter the First Player Name:'
+first_player_name = gets.chomp
+puts 'Enter the Second Player Name:'
+second_player_name = gets.chomp
+
+puts 'Lets start'
+
+puts '+---+---+---+'
+puts '| 1 | 2 | 3 |'
+puts '+---+---+---+'
+puts '| 4 | 5 | 6 |'
+puts '+---+---+---+'
+puts '| 7 | 8 | 9 |'
+puts '+---+---+---+'
+
+puts "First Player is #{first_player_name} and starts with X"
+
+puts 'Please select a avaiable cell from the board'
+move = gets.chomp.to_i
+
+while move.zero?
```

Comment:
[MANDATORY] 
- This while loop is just to validate if the move is valid, not the main while loop which keeps the game on play.
- Also, if a player puts -1 or 10 as a move it doesn't invalidate it because move.zero? condition will only invalidate the number 0 and other non-number characters. But, you know players are allowed to pick from 1 to 9 only.